code/modules/clothing/spacesuits/rig.dm

```diff
@@ -189,15 +189,11 @@
 		to_chat(H, "<span class='notice'>You deploy your hardsuit helmet, sealing you off from the world.</span>")
 	H.update_inv_head()
 
-/obj/item/clothing/suit/space/rig/attackby(obj/item/W as obj, mob/user as mob, params)
-	if(!isliving(user))
+/obj/item/clothing/suit/space/rig/attackby(obj/item/W as obj, mob/living/user as mob, params)
+	if(!istype(user))

```

Comment:
`if(!isliving(user)`

might as well kill off the `as obj` and `as mob` things too.